lldb/test/Shell/Swift/caching.test

```diff
@@ -0,0 +1,27 @@
+# REQUIRES: swift
+# REQUIRES: system-darwin
+
+# RUN: rm -rf %t && mkdir %t
+# RUN: split-file %s %t
+# RUN: %target-swiftc -g -Onone -save-temps \
+# RUN:          -module-cache-path %t/cache %t/main.swift \
+# RUN:          -cache-compile-job -cas-path %t/cas -explicit-module-build \
+# RUN:          -module-name main -o %t/main
+
+# RUN: %lldb %t/main -s %t/lldb.script
```

Comment:
Should there be some kind of FileCheck test?

Reply:
I can't figure out a good check for this because all I care about is the SwiftASTContext is created from the `expr` command.

Reply:
maybe you can run `log enable lldb types`, and then `CHECK: LogConfiguration`